hyprtester/test.conf

```diff
@@ -332,3 +332,5 @@ gesture = 3, down, mod:ALT, float
 
 gesture = 3, horizontal, mod:ALT, workspace
 
+windowrule = float, pin, class:kitty_C, size 200 200
```

Comment:
I dont mind but this I would mind. It's confusing as shit, I'd rather the order _must_ be `(rules), (selectors)`

Reply:
considering we split by a comma I don't see how this is a pain? Just go through them one by one

Reply:
c-f fixed on main, rebase for the CI to pass